src/containerapp/HISTORY.rst

```diff
@@ -0,0 +1,20 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.2
+++++++
+* Various fixes for bugs found
+* Dapr subgroup
+* Managed Identity
+
+0.1.1
+++++++
+* Various fixes for az containerapp create, update
+* Added github actions support
+* Added subgroups for ingress, registry, revision, secret
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
Why does the new extension need to release three versions?

Reply:
Changed to one version